pkg/code_coverage/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: code_coverage
```

Comment:
Please give this package a `README.md` explaining what this does... and how to invoke it :)